docs/source/howto/share_data.rst

```diff
@@ -155,9 +155,45 @@ Like all ``verdi`` commands, you can select a different AiiDA profile via the ``
 Querying the REST API
 ^^^^^^^^^^^^^^^^^^^^^
 
-Open the base URL of the REST API (by default: `http://127.0.0.1:5000/api/v4 <http://127.0.0.1:5000/api/v4>`_) in your web browser in order to get an overview of the available endpoints.
+A URL to query the REST API consists of:
 
-For an extensive user documentation, see the :ref:`AiiDA REST API reference <reference:rest-api>`.
+
+    1. The *base URL*, by default:
+
+        http://127.0.0.1:5000/api/v4
+
+        Querying the base URL returns a list of all available endpoints.
+
+    2. The *path* defining the requested *resource*, optionally followed by a more specific *endpoint*. For example::
+
+            /nodes
+            /nodes/page/2
+            /nodes/projectable_properties
+            /nodes/<uuid>
+            /nodes/<uuid>/links/outgoing
+
+       If no endpoint is appended, the API returns a list of objects of that resource.
+       In order to request a specific object of a resource, append its *UUID*.
+
+       .. note::
+
+           As usual, you can use partial UUIDs as long as they are unique.
+
+           In order to query by *PK* you need to use the ``id`` filter (see below).
+           This also applies to :py:class:`~aiida.orm.User` s, which don't have UUIDs.
+
+
+    3. (Optional) The *query string* for filtering, ordering and pagination of results. For example::
+
+        ?limit=20&offset=35
+        ?id=200
+        ?node_type=like="data%"
```

Comment:
This whole section should be de-indented. Also check the indentation of the parts, I believe the line `Querying the base URL ...` is prepended with one space too many?

Reply:
Thanks for noticing, will fix!